barber-compiler/build.gradle

```diff
@@ -5,6 +5,7 @@ sourceCompatibility = rootProject.ext.sourceCompatibilityVersion
 targetCompatibility = rootProject.ext.targetCompatibilityVersion
 
 dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Ah I see...that's probably why it's not on maven. So I just do `element.getAnnotation(Class.forName("android.support.annotation.DrawableRes"))` instead of `element.getAnnotation(DrawableRes.class)`??